### PR TITLE
feat(org-stats-8): Add selector for date range

### PR DESCRIPTION
**Pull request by leedongwei**
image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx [hunk 1]
```diff
@@ -61,8 +61,8 @@ export const CHART_OPTIONS_DATA_TRANSFORM: SelectValue<ChartDataTransform>[] = [
     disabled: false,
   },
   {
-    label: t('Day-to-Day'),
-    value: ChartDataTransform.DAILY,
+    label: t('Periodic'),
```
Comment (JoshFerge) [inline src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx:64]: In the subscription chart we call it "Day-to-day" -- should it be standardize?

Comment (JoshFerge) [reply src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx:64]: or is this different than day-to-day?

Comment (leedongwei) [reply src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx:64]: For subscription, a bar is always equivalent to 1 day. For usage stats, a bar could represent 1 hour, 4 hours, etc. 

`periodic` is an alright 1-word description for that, but I can change it if there's something better

Comment (JoshFerge) [reply src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx:64]: ah right. okay sounds good

**Review by markstory — APPROVED** [review]
Looks good to me

**Pull request merged**